src/App.js

```diff
@@ -22,6 +22,7 @@ import ThemeProvider from './styles/themes/ThemeProvider';
 import ThemeStylesProvider from './styles/ThemeStylesProvider';
 import {CurrentReportIDContextProvider} from './components/withCurrentReportID';
 import {EnvironmentProvider} from './components/withEnvironment';
+import 'react-native-url-polyfill/auto';
```

Comment:
Imported the polyfill in the root so that it can be utilized anywhere.This will override 2 classes from RN default i.e `URL` and `URLSearchParams`.

Should work without any problems but the import can also be moved `src/libs/Url.js` if required during review.